single_cell/utils/csvutils.py

```diff
@@ -392,10 +391,27 @@ def annotate_csv(infile, annotation_data, outfile, on="cell_id", write_header=Tr
     output.write_df(metrics_df)
 
 
+def filter_for_filesize(files, size=0):
```

Comment:
just catch the exception and continue

```
try:
   raw_data = pd.read_csv(filepath)
except pandas.io.common.EmptyDataError:
  continue
```

Its cleaner, less code and you dont have to guess the file sizes of empty gzips